### PR TITLE
Remove uses of GET_PEEP macro and replace with GetEntity

### PR DESCRIPTION
**Pull request by duncanspumpkin**
This simplifies the code a whole bunch and reduces the chances that there is operations on the wrong types.

### CODE REVIEW

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 1]
```diff
@@ -469,12 +469,14 @@ static int32_t cc_staff(InteractiveConsole& console, const arguments_t& argv)
...
 
-                if (int_valid[0] && int_valid[1] && ((GET_PEEP(int_val[0])) != nullptr))
+                if (int_valid[0] && int_valid[1] && ((GetEntity<Peep>(int_val[0])) != nullptr))
                 {
-                    Peep* peep = GET_PEEP(int_val[0]);
-
-                    peep->Energy = int_val[1];
-                    peep->EnergyTarget = int_val[1];
+                    Peep* peep = GetEntity<Peep>(int_val[0]);
+                    if (peep != nullptr)
```
Comment (Gymnasiast) [inline src/openrct2/interface/InteractiveConsole.cpp:475]: This will never happen, as you already did this check in the `if` on line 472.

Comment (duncanspumpkin) [reply src/openrct2/interface/InteractiveConsole.cpp:475]: Yeah but we will get nullptr deref compiler warnings otherwise

Comment (Gymnasiast) [reply src/openrct2/interface/InteractiveConsole.cpp:475]: Then we should take it out of the if on line 472.

Comment (duncanspumpkin) [reply src/openrct2/interface/InteractiveConsole.cpp:475]: Urgh sorry so tired today. Changed!

**Pull request merged**